tests/Unit/Helper/MiscHelperTest.php

```diff
@@ -33,4 +33,12 @@ public function getInputOutputForSplitSecondsMicroseconds()
             array('3.123456', array(3, 123456)),
         );
     }
+
+    public function testHexDump()
+    {
+        $res = MiscHelper::hexdump('FM', 'hmtlOutput' == false, 'uppercase' == false, 'return' == true);
```

Comment:
Yeah, I know that it works, but It's super confusing and OS libs should IMHO contain as little as possible hacks of this type.